app/components/app-monitoring/table/store-version-column/index.ts

```diff
@@ -0,0 +1,70 @@
+import { inject as service } from '@ember/service';
```

Comment:
look at line breaks between blocks

Reply:
The method of calculating statuses was moved to a helper so this would be discarded at a later PR.